src/Bridges/Variable/bridges/rsoc_to_psd.jl

```diff
@@ -248,19 +248,41 @@ function MOI.get(
     return dual
 end
 
+function MOI.supports(
+    model::MOI.ModelLike,
+    attr::MOI.VariablePrimalStart,
+    ::Type{<:RSOCtoPSDBridge},
+)
+    return MOI.supports(model, attr, MOI.VariableIndex)
+end
+
 function MOI.get(
     model::MOI.ModelLike,
-    attr::MOI.VariablePrimal,
+    attr::Union{MOI.VariablePrimal,MOI.VariablePrimalStart},
     bridge::RSOCtoPSDBridge,
     i::MOI.Bridges.IndexInVector,
 )
     value = MOI.get(model, attr, _variable(bridge, i))
-    if i.value == 2
+    if value !== nothing && i.value == 2
         return value / 2
     end
     return value
 end
 
+function MOI.set(
+    model::MOI.ModelLike,
+    attr::MOI.VariablePrimalStart,
+    bridge::RSOCtoPSDBridge,
+    value,
+    i::MOI.Bridges.IndexInVector,
+)
+    if value !== nothing && i.value == 2
+        value = 2 * value
+    end
+    MOI.set(model, attr, _variable(bridge, i), value)
```

Comment:
If `i.value == 2`, you need to set it for the whole diagonal. `_variable` only give you the first one since it is enough for `get`.

Reply:
Fixed